backend/src/main/java/nl/itvitae/ocaapp/tag/Tag.java

```diff
@@ -0,0 +1,22 @@
+package nl.itvitae.ocaapp.tag;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+public class Tag {
+
+    @Id
+    private Long id;
+    private String tagName;
```

Comment:
tagName can be renamed to "name", it's already obvious it is a tag name because it is part of the Tag entity/class